packages/@aws-cdk/aws-glue-alpha/README.md

```diff
@@ -221,7 +221,25 @@ new glue.Table(this, 'MyTable', {
 });
 ```
 
-By default, an S3 bucket will be created to store the table's data and stored in the bucket root. You can also manually pass the `bucket` and `s3Prefix`:
```

Comment:
Is this line removed on purpose? If so, why?

Reply:
yes, on purpose, this line is duplicated above